src/client/app/avdelingsleder/reservasjoner/components/ReservasjonerTabell.tsx

```diff
@@ -114,90 +121,97 @@ const ReservasjonerTabell: FunctionComponent<OwnProps & WrappedComponentProps> =
+			{reservasjonerSomSkalVises?.length > 0 && finnesSokResultat && (
 				<Table headerTextCodes={headerTextCodes} noHover>
 					{reservasjonerSomSkalVises.map((reservasjon) => (
-						<>
-							<TableRow
-								key={reservasjon.oppgaveId}
-								onMouseDown={() => velgReservasjon(reservasjon)}
-								onKeyDown={() => velgReservasjon(reservasjon)}
-							>
-								<TableColumn>{reservasjon.reservertAvNavn}</TableColumn>
+						<React.Fragment
+							key={`${reservasjon.oppgaveNøkkel.oppgaveEksternId} ${reservasjon.saksnummer} ${reservasjon.journalpostId}`}
+						>
+							<TableRow onMouseDown={() => velgReservasjon(reservasjon)} onKeyDown={() => velgReservasjon(reservasjon)}>
+								<TableColumn>{reservasjon.reservertAv}</TableColumn>
 								<TableColumn>{reservasjon.saksnummer}</TableColumn>
 								<TableColumn>
-									{getKodeverknavnFraKode(reservasjon.behandlingType, kodeverkTyper.BEHANDLING_TYPE, alleKodeverk) +
-										(reservasjon.tilBeslutter ? ' - [B] ' : '')}
+									{getKodeverknavnFraKode(
+										reservasjon.behandlingstype.kode,
+										kodeverkTyper.BEHANDLING_TYPE,
+										alleKodeverk,
+									) + (reservasjon ? ' - [B] ' : '')}
 								</TableColumn>
 								<TableColumn>
 									<FormattedMessage
 										id="ReservasjonerTabell.ReservertTilFormat"
-										values={getDateAndTime(reservasjon.reservertTilTidspunkt)}
+										values={getDateAndTime(reservasjon.reservertTil)}
 									/>
 								</TableColumn>
 								<TableColumn>
 									<Chevron
-										key={reservasjon.oppgaveId}
-										type={valgtReservasjon && valgtReservasjon.oppgaveId === reservasjon.oppgaveId ? 'opp' : 'ned'}
+										type={
+											valgtReservasjon &&
+											valgtReservasjon.oppgaveNøkkel.oppgaveEksternId === reservasjon.oppgaveNøkkel.oppgaveEksternId
+												? 'opp'
+												: 'ned'
+										}
 										className={styles.chevron}
 									/>
 								</TableColumn>
 							</TableRow>
-							{valgtReservasjon && valgtReservasjon.oppgaveId === reservasjon.oppgaveId && (
-								<Row className={styles.actionMenu}>
-									<Row>
-										<div className={styles.menuLine}>
-											<Image src={arrowIcon} className={styles.icon} />
-											<div
-												id="leggTilbake"
-												tabIndex={0}
-												className={styles.action}
-												role="button"
-												onClick={() => setShowOpphevReservasjonModal(true)}
-												onKeyDown={() => {
-													setShowOpphevReservasjonModal(true);
-												}}
-											>
-												<FormattedMessage id="ReservasjonerTabell.LeggTilbake" />
+							{valgtReservasjon &&
+								valgtReservasjon.oppgaveNøkkel.oppgaveEksternId === reservasjon.oppgaveNøkkel.oppgaveEksternId && (
+									<Row className={styles.actionMenu}>
+										<Row>
+											<div className={styles.menuLine}>
+												<Image src={arrowIcon} className={styles.icon} />
+												<div
```

Comment:
Kan du fikse denne til å være en `button` i samme slengen? Ser det er noe gammelt som bare har blitt flyttet

Reply:
Det er vel dette som må legges til som styling om du gjør det, med mindre det overskriver noe av eksisterende styling:
`padding: 0;
  border: none;
  background: none;

Reply:
![image](https://github.com/navikt/k9-los-web/assets/25080417/02c70211-d3cc-4453-9030-9e06e982e51e)

jeg lurer på om disse pilene betydde noe back in the day

Reply:
Jeg tror jeg tar dette i en egen branch, så bytter jeg ut tabellen med aksel også